news.html

```diff
@@ -54,6 +416,30 @@
       <!-- ############# Header ############# -->
       <header class="header_container nav-h">
         <div class="img">
+
+        <img id="rd-logo"  style="width: 90px; height: auto;" src="../images/rd.png"> 
+      </div>
+
+      </div>
+      <nav class="nav_menu">
+
+        <ul class="nav_link">
+          <li><i class="bi bi-house-door-fill"></i><a href="index.html">Home</a></li>
+          <li><i class="bi bi-info-circle-fill"></i><a href="index.html#about">About</a></li>
+          <li><i class="bi bi-newspaper"></i><a href="#">News</a></li>
+          <i class="bi bi-gear"></i><a href="dex.html">Service</a>
+          <li><i class="bi bi-person"></i><a href="index.html#patient-portal">Patient Portal</a></li>
+          <li><i class="bi bi-person-circle"></i><a href="index.html#doctors">Doctor</a></li>
+          <li><i class="bi bi-telephone"></i><a href="index.html#contact">Contact us</a></li>
+          <li><a href="#feedback"><i class="bi bi-star"></i>Rate Us</a></li>
+          <li><a href="login.html"><i class="bi bi-box-arrow-right"></i> Sign in/Sign up</a></li>
+          <a href="./Html-Files/Doctor Experience.html"><button>Find a Doctor</button></a>
+
+        </ul>
+      </nav>
+      </div>
+    </header>
+
           <img id="rd-logo" style="width: 90px; height: auto;" src="images/rd.png">
         </div>
         <nav class="nav_menu">
```

Comment:
Here's the duplicate code. See from line 422 to down has navbar code again same as above.